backend/src/diaries/diaries.repository.ts

```diff
@@ -156,14 +156,20 @@ export class DiariesRepository extends Repository<Diary> {
 
     return documents.hits.hits.map((hit) => hit._source as SearchDiaryDataForm);
   }
-  
-  findDiaryByTag(userId: number, tagName: string) {
+
+  findDiaryByTag(userId: number, tagName: string, lastIndex: number) {
```

Comment:
```suggestion 
  findDiaryByTag(userId: number, tagName: string, lastIndex: number | undefined) {
```
다른 부분에서 모두 undefined인 경우 `number | undefined` 같은 형식으로 명시해 주던데 하나로 통일하면 어떨까요?
전 없어도 괜찮은 것 같기도 하네요??

Reply:
넵 여기 통일하는게 좋겠네요! 음 저는 여기서 undefined가 예상치 못하게 들어오는 값이 아니라 들어올 것이 예상되는 타입이라 같이 명시해주는게 좋다고 생각했어요. 아래 if문에서도 lastIndex의 값이 있는지에 따라 추가되는 로직도 있으니까요!

Reply:
다른 부분도 통일해서 머지하겠습니다~